### PR TITLE
we create main view

### PR DESCRIPTION
**Pull request by JonathanAmador12**
main view is finished, i'd you to check out it

### CODE REVIEW

File: PokedexV2/Views/PokemonView.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//  PokedexV2
+//
+//  Created by JonathanA on 12/10/22.
+//
+
+import SwiftUI
+
+struct PokemonView: View {
+    
+    let formaGrid = [
```
Comment (AmadorHeE) [inline PokedexV2/Views/PokemonView.swift]: Change the name to gridForm

File: PokedexV2/Views/MainView.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+struct MainView: View {
+    var body: some View {
+        ZStack {
+            Color("primary")
+                .ignoresSafeArea()
+            VStack {
+                HStack {
+                    Text("Pokedex")
+                        .foregroundColor(.white)
+                    .fontWeight(.black)
```
Comment (AmadorHeE) [inline PokedexV2/Views/MainView.swift:20]: Add .font(.title)

**Review by AmadorHeE — CHANGES_REQUESTED** [review]

File: PokedexV2/Views/PokemonView.swift (hunk 1 above)
Comment (JonathanAmador12) [reply PokedexV2/Views/PokemonView.swift]: it is done

File: PokedexV2/Views/MainView.swift (hunk 2 above)
Comment (JonathanAmador12) [reply PokedexV2/Views/MainView.swift:20]: it's done

**Review by AmadorHeE — APPROVED** [review]
LGTM

**Pull request merged**